test/specs/parsing/registerCustomXPathFunction.tests.ts

```diff
@@ -809,7 +807,7 @@ describe('registerCustomXPathFunction', () => {
 			chai.assert.throws(
 				() =>
 					registerCustomXPathFunction(
-						'test-prefix-unique-to-unit-tests:func',
+						'a-random-prefix-to-prevent-collisions:func',
```

Comment:
Sure.

Reply:
Not really sure what to make of this one.

Reply:
It's changing one random prefix to another. Fine.